.github/actions/frontend/npm/publish/action.yml

```diff
@@ -0,0 +1,23 @@
+name: Publishes a frontend npm package
+
+inputs:
+  auto_token:
+    description: 'Used for publishing the GitHub release and creating labels'
+    required: true
+    type: string
+
+runs:
+  using: "composite"
+  steps:
+    - name: npm_publish_inputs
+      shell: bash
+      run: |
+        echo "auto_token=${{ inputs.auto_token }}"
+    - name: add_auto_to_globals
+      run: npm install -g auto@10.37.4
+      shell: bash
+    - name: deploy_to_npm
+      env:
+        GH_TOKEN: ${{ inputs.auto_token }}
+      run: npx auto shipit
```

Comment:
I am wondering if we should use npx -p package mode here and skip the above install

Reply:
Yeah I think that's a good call, updating now in the new PR